app/javascript/packages/document-capture/context/failed-capture-attempts.tsx

```diff
@@ -124,6 +139,10 @@ function FailedCaptureAttemptsContextProvider({
     setFailedSubmissionImageFingerprints(failedOnes);
   }
 
+  function onFailedCameraPermissionAttempt() {
```

Comment:
I did it this way to match the patterns in this file, could just pass the `increment...` function, but that feels worse to me.

Reply:
> I did it this way to match the patterns in this file, could just pass the `increment...` function, but that feels worse to me.

I think this is fine as-is. In support of passing it directly, the only concern I'd usually have with passing the original function is if there's a mismatch of expectations of what parameters might be passed through, which TypeScript should handle on our behalf as long as the `onFailedCameraPermissionAttempt` context type is properly defined.